test_data/makedata_suites/020_foxx.js

```diff
@@ -96,13 +96,39 @@ const crudTestServiceSource = {
 };
 
 (function () {
+  let aardvarkRoute = '/_db/_system/_admin/aardvark/index.html';
   let shouldValidateFoxx;
+  const onlyJson = {
+    'accept': 'application/json',
+    'accept-content-type': 'application/json'
+  };
+  let testFoxxReady = function(route) {
+    for (let i = 0; i < 200; i++) {
+      try {
+        reply = arango.GET_RAW(route, onlyJson);
+        if (reply.code === 200) {
```

Comment:
*should* be safe to test for 503 as well and throw if it's neither of those two status codes because that shouldn't happen, I think.

Reply:
ok, since it will just print and continue iterating I guess thats not a big change then...